lib/configs/browser.js

```diff
@@ -10,6 +10,9 @@ module.exports = {
     'github/no-blur': 'error',
     'github/no-dataset': 'error',
     'github/no-innerText': 'error',
-    'github/unescaped-html-literal': 'error'
+    'github/unescaped-html-literal': 'error',
+    'github/no-useless-passive': 'error',
+    'github/no-non-passive-high-frequency-events': 'error',
+    'github/prefer-observers-over-events': 'error'
```

Comment:
Do we consider this to be a breaking change? Do we want to release this as semver major?

Reply:
To answer my own question, precedent exists for this to be semver minor, see https://github.com/github/eslint-plugin-github/pull/80 which was released as `3.3.0`